src/common/components/select-repositories-list/index.js

```diff
@@ -0,0 +1,116 @@
+import React, { Component, PropTypes } from 'react';
```

Comment:
I've split out this file from `repositories-list` instead of modifying the existing file. It means I can continue working on this feature without breaking staging, and it should make integrating work a bit easier.

We are also coming to the point where we have two distinct repository lists:

- One for selecting repositories to be built
- One for reviewing repositories that are being built